### ISSUE TITLE
export-name throwing an error

### ISSUE THREAD

**Issue by cyberhck** [Microsoft/tslint-microsoft-contrib#815]
### Bug Report

-   **`tslint-microsoft-contrib` version**: latest
-   **TSLint version**: latest
-   **TypeScript version**: latest
-   **Running TSLint via**: (pick one) CLI

#### TypeScript code being linted

```tsx
export * from "./MyClass";
```

with `tslint.json` configuration:

```json
{
  "rules": {
    "export-name": true
  }
}
```

#### Actual behavior
![image](https://user-images.githubusercontent.com/3640284/52518671-d4415c00-2c80-11e9-90a3-7254873d694c.png)
```
➜  tinka-client git:(master) ✗ npm run lint

> @crazy-factory/ts-module-boilerplate@ lint /mnt/c/Users/ngautam/git/tinka-client
> tslint -t stylish --project tsconfig.json './src/**/*.ts'

The 'export-name' rule threw an error in '/mnt/c/Users/ngautam/git/tinka-client/src/index.ts':
TypeError: Cannot read property 'elements' of undefined
    at getExportsFromStatement (/mnt/c/Users/ngautam/git/tinka-client/node_modules/tslint-microsoft-contrib/exportNameRule.js:97:26)
    at ExportNameWalker.visitSourceFile (/mnt/c/Users/ngautam/git/tinka-client/node_modules/tslint-microsoft-contrib/exportNameRule.js:112:35)
    at ExportNameWalker.SyntaxWalker.visitNode (/mnt/c/Users/ngautam/git/tinka-client/node_modules/tslint/lib/language/walker/syntaxWalker.js:482:22)
    at ExportNameWalker.SyntaxWalker.walk (/mnt/c/Users/ngautam/git/tinka-client/node_modules/tslint/lib/language/walker/syntaxWalker.js:24:14)
    at Rule.AbstractRule.applyWithWalker (/mnt/c/Users/ngautam/git/tinka-client/node_modules/tslint/lib/language/rule/abstractRule.js:31:16)
    at Rule.apply (/mnt/c/Users/ngautam/git/tinka-client/node_modules/tslint-microsoft-contrib/exportNameRule.js:27:21)
    at Linter.applyRule (/mnt/c/Users/ngautam/git/tinka-client/node_modules/tslint/lib/linter.js:209:29)
    at /mnt/c/Users/ngautam/git/tinka-client/node_modules/tslint/lib/linter.js:150:85
    at Object.flatMap (/mnt/c/Users/ngautam/git/tinka-client/node_modules/tslint/lib/utils.js:160:29)
    at Linter.getAllFailures (/mnt/c/Users/ngautam/git/tinka-client/node_modules/tslint/lib/linter.js:150:32)
```

#### Expected behavior
tslint to pass

Did I do something wrong? My project is almost empty, I'm starting from scratch, and it used to be okay before.

I'm sure it's because of `export-name` because it says so on error message, and if I set the rule to false, it just passes.

**Comment by cyberhck** [comment]
here's my state of project, it's basically empty:
![image](https://user-images.githubusercontent.com/3640284/52518685-17033400-2c81-11e9-89f9-3b434f0a75c9.png)

![image](https://user-images.githubusercontent.com/3640284/52518687-21bdc900-2c81-11e9-87f0-f01cd204998a.png)

**Comment by IllusionMH** [comment]
Looks like duplicate of #664.

**Comment by cyberhck** [comment]
woops, sorry about that 😄
